javascript/chronometer.js

```diff
@@ -1,26 +1,39 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+      this.currentTime = 0;
+      this.intervalId = 0;
+   }
+
   startClick(callback) {
-    // ... your code goes here
+    let intervalID = setInterval(() => {
```

Comment:
You are saving the interval ID into a simple variable, but not in the intervalID property (this.intervalId). So when you call the clearInterval in line 31, it is never clearing this interval, it is just trying to clear an interval whose id is 0.